inference/statnode.py

```diff
@@ -1,110 +1,215 @@
+    name : string
+        NBA player for stat retrieval
+    stat_val: string
+        statistic value
+    
+    Returns
+    -------
+    resp_list : list
+        The list of random response
+    """
+    def generate_random_response(self, name, stat, stat_val):
+        resp_1 = "Seems {} has an {} out of {}.".format(name, stat, stat_val)
+        resp_2 = "After checking my little black book, I've found that {} has {} under his name for {}.".format(name, stat_val, stat)
+        resp_3 = "{}...obviously".format(stat_val)
+        resp_4 = "Last I checked... {} had a {} out of {}.".format(name, stat, stat_val)
+        resp_5 = "Do you not have every NBA stat since the beginning of time memorized? Well, he has a {} out of {}.".format(stat, stat_val)
+        resp_6 = "{} has {} out of {}, impressive isn't it?".format(name, stat, stat_val)
+        resp_7 = "As we are speaking, {} has {} out of {}".format(name, stat, stat_val)
+        resp_8 = "After watching hours of tapes, I came up with {} out of {} for {}".format(stat, stat_val, name)
+        resp_9 = "If my calculations are correct {} has {} out of {} during his career".format(name, stat, stat_val)
+        resp_10 = "Couldn't you Google that? Well since we're already here: {} has {} out of {}".format(name, stat, stat_val)
+        resp_list = [resp_1,
+                     resp_2,
+                     resp_3,
+                     resp_4,
+                     resp_5,
+                     resp_6,
+                     resp_7,
+                     resp_8,
+                     resp_9,
+                     resp_10]
+        return random.choice(resp_list)
+
+    """
+        Function to generate a response
+        
+        Parameters
+        ----------
+        self : none
+            
+        
+        Returns
+        -------
+        name : string
+          NBA player for stat retrieval
+        stat : string
+          The stat that is being compared to stat_val
+        stat_val : string
+          statistic value
+    """
+    def response(self):
+        name = self.extract_name()
+        stat = self.extract_stat()
+        if not name:
+            return inc_name
+        elif not stat:
+            return inc_stat
+        stat_val = self.get_player_stat(name, stat)
+        return self.generate_random_response(name, stat, stat_val)
+
+    """
+        Function that extracts the player's name
+        
+        Parameters
+        ----------
+        self : none
+            
+        Returns
+        -------
+        name : string
+          NBA player for stat retrieval
+    """
+
+    def extract_name(self):
+        doc = self.nlp(self.query)
+        name = None
+        for entity in doc.ents:
+            if entity.label_ == "PERSON":
+                name = entity.text
+        if not name:
+            for entity in doc.ents:
+                if entity.label_ == "ORG":
+                    name = entity.text
+
+        return name
+  <<<<<<< Florent18570
```

Comment:
This line is throwing the error. If you remove it the CI tests should pass :)

Reply:
Okay I found out why the check didn't work :)